typescript/codepipeline-build-deploy/lib/codepipeline-build-deploy-stack.ts

```diff
@@ -69,22 +86,30 @@ export class CodepipelineBuildDeployStack extends cdk.Stack {
           TASK_DEFINITION_ARN: { value: fargateTaskDef.taskDefinitionArn },
           TASK_ROLE_ARN: { value: fargateTaskDef.taskRole.roleArn },
           EXECUTION_ROLE_ARN: { value: fargateTaskDef.executionRole?.roleArn },
+          
         },
+   
       },
     });
     
+   
+
     // CodeBuild project that builds the Docker image
     const buildTest = new codebuild.Project(this, "BuildTest", {
       buildSpec: codebuild.BuildSpec.fromSourceFilename("buildspec.yaml"),
-      source: codebuild.Source.codeCommit({ repository: codeRepo }),
+      source: codebuild.Source.gitHub({ owner: 'owner_name', repo: 'repo_name', webhook: true }),
```

Comment:
Another location that needs to documented so that the user knows this value needs to be updated.